pytest_catchlog.py

```diff
@@ -139,10 +180,13 @@ def emit(self, record):
 class CatchLogFuncArg(object):
```

Comment:
I'd just call it `CatchLog` or maybe `LogCatcher`. Though pytest calls the object behind the `capsys`/`capfd` fixtures `CaptureFixture`, so `CatchLogFixture` or `LogCaptureFixture` would certainly work as well.